contracts/Penalizer.sol

```diff
@@ -26,6 +26,10 @@ contract Penalizer is IPenalizer{
 
     }
 
+    function relayManagerOnly(IRelayHub hub) private view {
```

Comment:
Call it 'require' so it is obvious this method reverts if not manager. 'xxOnly' is more of a modifier name.

Reply:
Converted to modifier then